Merge refactoring changes with master branch

kramsey458 opened the pull request:
Issue #32
Game is still running as it originally did. Nothing appears to be broke (yet)

kramsey458 committed:
Merge pull request #3 from StreakForHumanity/master

Repo update

kramsey458 committed:
Issue #32
Refactored code in StreakerGame.java into other classes.
Removed character class.
Created a constants class for constants to be accessed for multiple classes.
Created GraphicsController class to handle graphic manipulation of the game stage.
Moved methods around here and there to provide better organization to code structure.

kramsey458 committed:
Delete vcs.xml

ms2300 commented on src/Coin.java:
```diff
@@ -1,22 +1,25 @@
         this.MOVING_SPEED = MOVING_SPEED;
         this.setImage(_PATH);
         this.resetPosition();
+        this.constants = new Constants(background);
```
sConstants doesn's appear to be used in the coin clsss and moving speed is still there and being used. Refactor this please

Github is fucking up my spelling I'm so sorry I can't fix this

ms2300 commented on src/Constants.java:
```diff
@@ -0,0 +1,51 @@
+public class Constants {
```
Shouldn't all these methods be static? Why is there a constructor and non-static methodss

ms2300 commented on src/GraphicsController.java:
```diff
@@ -0,0 +1,25 @@
+import javafx.scene.canvas.GraphicsContext;
```
sThis is where the game will be yeahs

ms2300 commented on src/Streaker.java:
```diff
@@ -23,6 +26,38 @@ public Streaker(double FRAME_DURATION, int SCREEN_HEIGHT, BackgroundItem backgro
+        constants = new Constants(background);
+    }
+
+    public void handleCharacterPosition() {
```
Good shis

ms2300 reviewed (changes requested):
Looks good but have a couple questions

kramsey458 committed:
Issue #32
Made things static where they should be static. Addressed issues in notated by Matt in the pull request.
